### ISSUE TITLE
Fix docstrings in twisted.test.test_jelly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8671]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)| @acabhishek942 reported|
|-|-|
|Trac ID|trac#8671|
|Type|enhancement|
|Created|2016-07-25 10:06:46Z|

Docstrings in twisted.test.test_jelly is missing for some test and also has some typos

<details><summary>Searchable metadata</summary>

```
trac-id__8671 8671
type__enhancement enhancement
reporter__acabhishek942 acabhishek942
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1469441206682923 1469441206682923
changetime__1469536374556240 1469536374556240
version__None None
owner__acabhishek942 acabhishek942

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_1"></a>|@acabhishek942 commented|
|-|-|

See https://github.com/twisted/twisted/pull/409

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @acabhishek942|
|-|-|

All good. Thanks. Will merge!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 737203ec952a281a9fa21f5f146dff292eacf4dc
```
#!CommitTicketReference repository="" revision="737203ec952a281a9fa21f5f146dff292eacf4dc"
Merge acabhishek942/8671-acabhishek942-test-jelly-docstrings: Fix docstrings in t.test.test_jelly


Author: acabhishek942
Reviewers: rodrigc, adiroiban
Fixes: #8671
```
